project/assets/newevaluationStyle.less

```diff
@@ -0,0 +1,358 @@
+    border-bottom: 1px solid lightgrey;
+  }
+
+  .inputs-date {
+    width: 26%;
+    height: 2.7rem;
+    border: none;
+    font-size: 128%;
+    color: rgb(82, 86, 89);
+    border-bottom: 1px solid lightgrey;
+  }
+
+}
+
+
+
+
+/* -------------------------------Technical Level---------------------------- */
+
+.content-box:first-child {
+  background-color: whitesmoke;
+  height: 15rem;
+}
+
+table {
+  border-collapse: collapse;
+  width: 72rem;
+  height: 16%;
+  position:absolute;
+  margin-left: 21%;
+  margin-top: 2%;
+  text-align: center;
+
+}
+
+th {
+  background-color: #03A9F4;
+  color: white;
+  font-size: 136%;
+  font-weight: normal;
+  height: 3rem;
+}
+
+.second-half-of-table {
+  height: 4rem;
+  border-collapse:collapse
+
+}
+
+input[type=radio]{
+  color: lightblue;
+}
+
+td {
+  color: #4d4d4d;
+  //border: 0.5px solid black;
+  border-top: 1px solid black;
+  border-bottom: 1px solid black;
+  border-spacing: 0;
+}
+
+td:first-child {
+  border-left: 1px solid black;
+}
+
+td:last-child {
+  border-right: 1px solid black;
+}
+
+caption{
+  margin-left: -84%;
+  font-size: 1.6rem;
+  color: #03A9F4;
+  margin-bottom: 0.6rem;
+}
+
+
+/* ----------------------- Should the candidate be hired? ------------------- */
+
+.content-box:nth-child(2) {
+  background-color: whitesmoke;
+  margin-top: 2rem;
+  width: 100%;
+  height: 12rem;
+}
+
+h2 {
+  margin-left: 25.5rem;
+  background-color: whitesmoke;
+  font-size: 1.6rem;
+  color: #03A9F4;
+  margin-bottom: -2.5rem;
+  font-weight: normal;
+}
+
+
+textarea {
+  border: 1px solid grey;
+  width: 72rem;
+  height: 6.5rem;
+  margin-left: 21%;
+  margin-top: 2.5%;
+  font-style: italic;
+  background-color: whitesmoke;
+  font-size: 1.1rem;
+  color: grey;
+}
+
+
+/* -------------------------------General Impression------------------------- */
+
+.content-box:nth-child(3) {
+  background-color: whitesmoke;
+  margin-top: 2rem;
+  width: 100%;
+  height: 12rem;
+}
+
+
+
+/* ------------------------Workflow, Leadership & Soft skills-----------------*/
+
+.content-box:nth-child(4) {
+  background-color: whitesmoke;
+  margin-top: 2rem;
+  width: 100%;
+  height: 12rem;
+}
+
+
+/* -------------------------------OOP, Design Patterns------------------------*/
+
+.content-box:nth-child(5) {
+  background-color: whitesmoke;
+  margin-top: 2rem;
+  width: 100%;
+  height: 16rem;
+  padding-top: 0rem;
+}
+
+legend {
+  font-size: 1.5rem;
+  color: #03A9F4;
+}
+
+fieldset {
+  border: 1px solid black;
+  width: 70.5rem;
+  height: 67%;
+  margin-left: 25rem;
+  font-size: 1.3rem;
+  color: rgb(82, 86, 89);
+}
+
+li {
+  display: inline;
+  float: left;
+  list-style-type: none;
+}
+
+li {
+  width: 29%;
+  margin-left: -4%;
+  margin-right: 8%;
+  margin-bottom: 2%;
+}
+
+select {
+  float: right;
+}
+
+option {
+  width: 2%;
+}
+
+label {
+  width: 10%;
+}
+
+/* ------------------------------------ HTTP ---------------------------------*/
+
+.content-box:nth-child(6) {
```

Comment:
I'm seeing quite a bit of similarity between these `.content-box:nth-child(n)` selectors. Do you think that having a common class for all of them would help you with code reuse and not repeating the same properties for all?